src/calculateRentalCost.js

```diff
@@ -3,8 +3,23 @@
  *
  * @return {number}
  */
+
+const daysWithLowADiscount = 3;
+const daysWithLargeADiscount = 7;
+const rentOfEveryDay = 40;
+const lowDiscount = 20;
+const largeDiscount = 50;
+
 function calculateRentalCost(days) {
-  // write code here
+  if (days >= daysWithLowADiscount && days < daysWithLargeADiscount) {
+    return days * rentOfEveryDay - lowDiscount;
+  }
+
+  if (days >= daysWithLargeADiscount) {
+    return days * rentOfEveryDay - largeDiscount;
```

Comment:
The check for the larger discount is correct, and it's good that you've returned early, avoiding the need for an `else` statement.